autoload/ale/references.vim

```diff
@@ -67,17 +68,33 @@ function! ale#references#HandleTSServerResponse(conn_id, response) abort
 endfunction
 
 function! ale#references#FormatLSPResponseItem(response_item, options) abort
+    let l:line_text = ''
+
+    let l:line= a:response_item.range.start.line
+    let l:col = a:response_item.range.start.character
+    let l:filename = ale#util#ToResource(a:response_item.uri)
+
+    if get(a:options, 'show_contents') == 1
+        try
+            let l:line_text = substitute(readfile(l:filename)[l:line], '^\s*\(.\{-}\)\s*$', '\1', '')
```

Comment:
I've merged this, and thanks for this. An improvement we could try here is passing the `max` argument to readfile to read only up to the line we need.